.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,23 @@
+name: Feature Request
+description: Suggest an idea for Xournal+.
+labels: enhancement
+body:
+  - type: textarea
+    attributes:
+      label: Is Your Feature Request Related to a Problem?
+      description: A clear and concise description of what your problem is.
+      placeholder: Ex. I'm always frustrated when...
+  - type: textarea
+    attributes:
+      label: Feature Description
+      description: A clear and concise description of the feature that you would propose.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Alternatives That You've Considered
+      description: Add any alternative solutions that you may have considered here.
```

```suggestion
      label: Alternative Solutions
      description: Add any alternative solutions that you have considered here.
```

---

How about
```suggestion
      label: Alternative Solutions
      description: Add any alternative solutions that you may have considered here.
```
In my opinion, "may" is important language to use. It's not a guarantee that the individual has considered alternative solutions -- removing "may" makes it sound like one *must* think of alternatives.

---

I'm fine with your suggestion.